gen/kypo_portal

```diff
@@ -42,12 +40,10 @@ our $userStruc = {};
 our $u_status = {};
 our $u_email = {};
 our $u_name = {};
-our $u_eppn = {};
 our $u_lf_name = {};
 
 our $groupStruc = {};
 our $g_name = {};
-our $g_par_id = {};
 
 our $membershipStruc = {};
```

Comment:
Please use 'my' instead of 'our' for variables from u_status to membershipStruc.